tox.ini

```diff
@@ -8,23 +8,20 @@
 skipsdist = true
 envlist = py{26,27,33,34,35},py{26,27}-oldest,cover,lint
 
-# nosetest -v => more verbose output, allows to detect busy waiting
-# loops, especially on Travis
-
 [testenv]
-# packages installed separately to ensure that dowstream deps problems
+# packages installed separately to ensure that downstream deps problems
 # are detected, c.f. #1002
 commands =
-    pip install -e acme[testing]
-    nosetests -v acme
-    pip install -e .[testing]
-    nosetests -v letsencrypt
+    pip install -e acme
+    python acme/setup.py test
```

Comment:
I believe all lines in `tox.ini` should be reverted, including above 2, due to the fact we changed handling of install_requires and tests_requires. Especially given the fact that you removed `tests_require=install_requires` lines throughout.